apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/pac/McsClaimResponseTransformerV2.java

```diff
@@ -20,27 +20,31 @@
 import org.hl7.fhir.r4.model.Meta;
 import org.hl7.fhir.r4.model.Reference;
 import org.hl7.fhir.r4.model.codesystems.ClaimType;
+import org.springframework.stereotype.Component;
 
 /** Transforms FISS/MCS instances into FHIR {@link ClaimResponse} resources. */
+@Component
 public class McsClaimResponseTransformerV2 extends AbstractTransformerV2 {
 
+  /** The Metric registry. */
+  private final MetricRegistry metricRegistry;
+
   /** The metric name. */
-  private static final String METRIC_NAME =
-      MetricRegistry.name(McsClaimResponseTransformerV2.class.getSimpleName(), "transform");
+  private final String METRIC_NAME;
 
   /**
    * There are only 2 statuses currently being used, and only the ones listed below are mapped to
    * "CANCELLED". For brevity, the rest are defaulted to "ACTIVE" using {@link
    * Map#getOrDefault(Object, Object)}.
    */
-  private static final Map<String, ClaimResponse.ClaimResponseStatus> STATUS_MAP =
+  private final Map<String, ClaimResponse.ClaimResponseStatus> STATUS_MAP =
       Map.of(
           "r", ClaimResponse.ClaimResponseStatus.CANCELLED,
           "z", ClaimResponse.ClaimResponseStatus.CANCELLED,
           "9", ClaimResponse.ClaimResponseStatus.CANCELLED);
 
   /** The known MCS codes and their associated {@link ClaimResponse.RemittanceOutcome} mappings. */
-  private static final Map<String, ClaimResponse.RemittanceOutcome> OUTCOME_MAP =
+  private final Map<String, ClaimResponse.RemittanceOutcome> OUTCOME_MAP =
       Map.ofEntries(
```

Comment:
Is there a reason to turn these constants into instance variables?

Reply:
Nope, overzealous removal of static items. Changed them back.